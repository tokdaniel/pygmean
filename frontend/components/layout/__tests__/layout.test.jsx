import React from 'react'
import renderer from 'react-test-renderer'

import Layout from '../'

describe('With Snapshot Testing', () => {
  it('Runs', () => {
    const component = renderer.create(<Layout />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
