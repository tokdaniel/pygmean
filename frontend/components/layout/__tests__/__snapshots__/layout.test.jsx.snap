// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`With Snapshot Testing Runs 1`] = `
<div>
  Something
</div>
`;
